freqtrade/exchange/exchange.py

```diff
@@ -716,17 +771,28 @@ def fetch_dry_run_order(self, order_id) -> Dict[str, Any]:
 
     # Order handling
 
+    def lev_prep(self, pair: str, leverage: float):
+        self.set_margin_mode(pair, self.collateral)
+        self._set_leverage(leverage, pair)
+
+    def _get_params(self, time_in_force: str, ordertype: str, leverage: float) -> Dict:
+        params = self._params.copy()
+        if time_in_force != 'gtc' and ordertype != 'market':
+            param = self._ft_has.get('time_in_force_parameter', '')
+            params.update({param: time_in_force})
+        return params
+
     def create_order(self, pair: str, ordertype: str, side: str, amount: float,
-                     rate: float, time_in_force: str = 'gtc') -> Dict:
+                     rate: float, time_in_force: str = 'gtc', leverage=1.0) -> Dict:
 
         if self._config['dry_run']:
             dry_order = self.create_dry_run_order(pair, ordertype, side, amount, rate)
```

Comment:
I would want to see support for leverage also for dry-run (if you want as seperate, follow-up PR to this).

otherwise testing the pure logic (without the exchange sides) will become a costly endavor.